core/trino-main/src/test/java/io/trino/block/TestBlockAssertions.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.block;
+
+import org.testng.annotations.Test;
+
+import java.util.Objects;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import static io.trino.block.BlockAssertions.generateListWithNulls;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.testng.Assert.assertEquals;
+
+public final class TestBlockAssertions
+{
+    @Test
+    public void generatesListWithNulls()
+    {
+        // when
+        Iterable<Integer> list = generateListWithNulls(10, 0.2F, () -> 1);
+
+        // then
+        assertEquals(stream(list).filter(Objects::isNull).count(), 2);
+        assertEquals(stream(list).filter(o -> !(o == null)).count(), 8);
+    }
+
+    @Test
+    public void failsIFPositionCountToLow()
+    {
+        // when
+        // then
+        assertThatThrownBy(() -> generateListWithNulls(4, 0.2F, () -> 1)).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    private static <T> Stream<T> stream(Iterable<T> list)
+    {
+        return StreamSupport.stream(list.spliterator(), false);
```

Comment:
I am pretty sure that Guava have this method somewhere

Reply:
Indeed it has. WIll use it.